e2e/app/component/bypass-modules.ts

```diff
@@ -0,0 +1,59 @@
+import { Page } from 'puppeteer';
+import BaseMenu from './base-menu';
+import Checkbox from 'app/element/checkbox';
+
+const defaultXpath = '//*[@id="popup-root"]';
+
+export default class BypassPopup extends BaseMenu {
+  constructor(page: Page, xpath: string = defaultXpath) {
+    super(page, xpath);
+  }
+
+  /**
+   *  Get texts of all visible options.
+   */
+  async getAllToggleTexts(): Promise<string[]> {
+    const selector = `${this.getXpath()}//label/span/text()`;
+    return this.getMenuItemTexts(selector);
+  }
+
+  getMenuItemXpath(toggleAccessText: string): string {
+    return `//*[@role="switch" and normalize-space()="${toggleAccessText}"]`;
+  }
+
+  getTrainingBypassToggle(): Checkbox {
```

Comment:
can we make these more generic?  I see they're identical except for the text

Reply:
Hey Joel, by generic, do you mean using the same functions for `getToggles` on the admin-table and admin-user-profile page?

Reply:
I mean we can use a single function here instead of 4.  Something like `getBypassToggle(Modules.COMPLIANCE)`

Reply:
got it..will try! Thanks!

Reply:
not updated this yet.